edenai_apis/apis/google/google_helpers.py

```
@@ -109,6 +109,52 @@
+        try:
+            message = exc.message
+        except:
+            message = str(exc)
+        if error_encoding_str in str(exc):
+            message = msg_exception_encoding
+        if any(str_error in str(exc) for str_error in wrong_job_id_strs):
+            raise AsyncJobException(
+                reason=AsyncJobExceptionReason.DEPRECATED_JOB_ID, code=status_code
+            )
+        raise ProviderException(message, code=status_code)
+    except Exception as exc:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)